lib/vmdb/deprecation.rb

```diff
@@ -1,7 +1,7 @@
 module Vmdb
   class Deprecation
     def self.instance
-      @instance ||= ActiveSupport::Deprecation.new("D-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
+      @instance ||= ActiveSupport::Deprecation.new("F-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
```

Comment:
Unfortunately I think it's too late for that, we would have had to change it before the Darga branch.  The current message looks like this:
```ruby
irb(main):001:0> Vmdb::Deprecation.deprecation_warning("ABC")
DEPRECATION WARNING: ABC is deprecated and will be removed from ManageIQ D-release (called from irb_binding at (irb):1)
```